packages/helpers/src/tests/get-unique-passcode.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { forces } from '@serge/mocks'
+import { ForceData } from '@serge/custom-types'
+import { getUniquePasscode } from '../'
+
```

Comment:
@sanjayradadiya  - you can probably delete this test. As you said, it's really hard to test for the creation of a unique passcode.

Ah no, let's keep it.  At least it's proving `getUniquePasscode` still works, without any compiler errors.

Actually, I suggest adding:
`expect(res).toBeTruthy()` 
- just to check _something_ is returned.